analysis_driver/notification.py

```diff
@@ -1,4 +1,59 @@
-from egcg_core import notifications
+from egcg_core import notifications, clarity
+from pyclarity_lims.entities import Step
+from analysis_driver.exceptions import PipelineError
+
+
+class LimsNotification():
+    def __init__(self, name):
+        self.name = name
+        self.sample_list = clarity.get_list_of_samples(self.name)
+        self.artifacts = [sample.artifact for sample in self.sample_list]
+        self.step = None
+
+    @property
+    def lims(self):
+        return clarity.connection()
+
+    @property
+    def postseqlab_stages(self):
+        return self.lims.get_workflows(name='PostSeqLab EG 1.0 WF')[0].stages
+
+    def get_stage(self, stage_name):
+        stage = [stage for stage in self.postseqlab_stages if stage == stage_name]
+        if not len(stage) == 1:
+            raise PipelineError('Can not find %s from Workflow Stages' % stage_name)
+        return stage[0]
+
+    def route_sample_to_data_processing(self):
+        self.lims.route_artifacts(self.artifacts, stage_uri=self.get_stage('Data Processing EG 1.0 ST').uri)
+
+    def create_step(self):
+        self.step = Step.create(self.lims, protocol_step=self.get_stage('Data Processing EG 1.0 ST').step, inputs=[self.artifacts], container_type_name='Tube')
```

Comment:
If `self.artifacts` is a list then you would pass a list of of one list to inputs.
Make sure the artifact is properly passed to `route_artifacts` and  `inputs`